flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/PipelinedRegionSchedulingStrategy.java

```diff
@@ -61,7 +62,8 @@ public class PipelinedRegionSchedulingStrategy implements SchedulingStrategy {
     private final Set<ConsumedPartitionGroup> crossRegionConsumedPartitionGroups =
```

Comment:
consumedPartitionGroupsOfRegion -> producedPartitionGroupsOfRegion

I think this is more accurate and can make the code a bit easier to understand. The documentation should be updated accordingly.

Reply:
good idea, changed.